packages/bonde-admin-canary/src/scenes/Logged/scenes/Chatbot/Root.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
```

Comment:
Adicionar o `Fragment` no destructuring no import do `React`
```suggestion
import React, { Fragment, useState } from 'react'
```